platform/ios/src/MGLMapView.mm

```diff
@@ -6313,6 +6413,7 @@ - (void)updateScaleBar
     if ( ! self.scaleBar.hidden)
     {
         [(MGLScaleBar *)self.scaleBar setMetersPerPoint:[self metersPerPointAtLatitude:self.centerCoordinate.latitude]];
+        [self installScaleBarConstraints];
```

Comment:
Placing this here can cause `installScaleBarConstraints` be to called many times a second (i.e., every time the camera changes and the scale is updated). Constantly updating the constraints in this way seems like it could have a negative effect on performance — can/should this be avoided?